test/cmd/policy.sh

```diff
@@ -80,8 +80,22 @@ os::cmd::expect_success_and_text 'oc policy can-i --list' 'get update.*imagestre
 os::cmd::expect_success_and_text 'oc policy can-i create pods --all-namespaces' 'yes'
 os::cmd::expect_success_and_text 'oc policy can-i create pods' 'yes'
 os::cmd::expect_success_and_text 'oc policy can-i create pods --as harold' 'no'
+os::cmd::expect_failure 'oc policy can-i create pods --as harold --user harold'
```

Comment:
Especially with the influx of these tests, `can-i` should have it's own suite in it's own file, not here.